openedx/core/djangoapps/user_authn/serializers.py

```diff
@@ -0,0 +1,57 @@
+"""
+MFE Context API Serializers
+"""
+
+from rest_framework import serializers
+
+
+class ContextDataSerializer(serializers.Serializer):
+    """
+    Context Data Serializers
+    """
+
+    currentProvider = serializers.CharField()
+    platformName = serializers.CharField()
+    providers = serializers.ListField()
+    secondaryProviders = serializers.ListField()
+    finishAuthUrl = serializers.CharField()
+    errorMessage = serializers.CharField()
+    registerFormSubmitButtonText = serializers.CharField()
+    syncLearnerProfileData = serializers.CharField()
+    countryCode = serializers.CharField()
+    pipelineUserDetails = serializers.DictField(source='pipeline_user_details')
```

Comment:
This looks fine but I think I have some questions here. 

1. why are the sources not provided for these fields?
2. do we only need serialization at this level? 
3. does not any field need serialization?
4. Did you confirm this from the frontend? 
5. or did you test this?